main.go

```diff
@@ -83,10 +92,12 @@ func main() {
 	// Calculate staging or production version
 	switch environment {
 	case "staging":
+		log.Println("Environment is staging")
 		finalTag := stagingVersion(commit.Tag, commit.Type, rc)
-		log.Println(finalTag)
-	case "main", "master":
-		finalTag := productionVersion(commit.Tag, rc)
-		log.Println(finalTag)
+		utils.SetTagOutputName(finalTag)
+	case "main", "master", "production":
+		log.Println("Environment is production")
+		finalTag := productionVersion(commit.Tag, commit.Type, rc)
```

Comment:
1. for both staging version and production version you can commit as the first arg in the function this way you wont need to pass commit.Tag and commit.Type only commit for example
productionVersion(commit, rc)
2. In production Commit Type should be hard coded to path (bugfix)

Reply:
1. - fixed
2. didn't know - fixed